### ISSUE TITLE
not able to build project branches

### ISSUE THREAD

**Issue by ric03uec** [Shippable/support#2540]
### Is your request:

- [ ] A feature request: The feature you want currently does not exist. If implemented, will provide additional value to you.
- [x] A technical issue: You are unable to use Shippable's service per your expectation & need help to resolve the issue.
- [ ] A question: A non-technical question such as billing.

### Description of your issue:

projects for `TodayTix` subscription are not running branch builds since 04/16. They're also getting the following error while navigating to runs page

![image](https://cloud.githubusercontent.com/assets/1110988/14613819/c87e8a4c-0563-11e6-8142-47bfcee696d2.png)

@MattGerstman

**Comment by jessestuart** [comment]
Hey @ric03uec,
Any chance we could get an ETA on a fix for this? We can fall back to a manual build process if necessary for the day, but it may not be worth the overhead if we can expect a fix within a few hours.

Let us know what you think!

cc: @MattGerstman

**Comment by ric03uec** [comment]
@jessestuart im still trying to narrow the problem down so its difficult for me to give an ETA right now. This seems like an isolated incident because of which its taking time to figure it out.
It would really help expedite the process if anyone from your team can get on a call with us. 
let me know if that works for you guys. thanks.

**Comment by jessestuart** [comment]
@ric03uec absolutely, I'm available to get on a call now. I'll email my number to the address listed on your github profile.

**Comment by jessestuart** [comment]
Thanks for hopping on a google hangout @ric03uec!

Quick recap in case others encounter a similar issue down the road:

In the end, what ended up resolving the issue for us was going to our Github account profile, selecting OAuth applications, then selecting `prod-shippable`:
<img width="668" alt="screenshot 2016-04-18 15 37 45" src="https://cloud.githubusercontent.com/assets/583147/14617731/55d726a6-057d-11e6-8db6-b005251abdeb.png">

At which point we discovered that we had the option to "Grant access" to our company repo, something we'd never seen before: (shown here after we'd already granted access for TodayTix)
<img width="667" alt="screenshot_2016-04-18_15_54_04" src="https://cloud.githubusercontent.com/assets/583147/14617910/200e6dda-057e-11e6-98f1-0451f5e1e4be.png">

After we did that, within a few minutes we were able to trigger builds again on all branches.

As far as we know, no one on our team had touched our github/shippable permissions, so it's a little unclear what caused this; but we're happy to be back up and running again!
